search/nearby_points_sweeper.cpp

```diff
@@ -0,0 +1,33 @@
+#include "search/nearby_points_sweeper.hpp"
+
+namespace search
+{
+// NearbyPointsSweeper::Event ----------------------------------------------------------------------
+NearbyPointsSweeper::Event::Event(Type type, double y, double x, size_t index)
```

Comment:
No, thanks. y is like a time (or priority) parameter. I can rename it, but I don't want to exchange it with x.